### PR TITLE
feat: MET-2525 add configuration support

### PR DESCRIPTION
**Pull request by dawidsdh**
This PR will add configuration support so the cherryGL can read it

### CODE REVIEW

File: src/assets/scripts/ProjectManager/Scenegraph.js [hunk 1]
```diff
@@ -391,16 +388,22 @@ module.exports = () => {
 
   const constructGraph = (opt) => {
     if (sceneprops.project.data == undefined) return;
-    const tree =
-      sceneprops.project.data['scene'][sceneprops.project.data.selected_scene]
-        .tree;
+    const scene =
+      sceneprops.project.data['scene'][sceneprops.project.data.selected_scene];
+    const tree = scene.tree;
+    const configurations = scene.configurations;
```
Comment (mwojcik-sdh) [inline src/assets/scripts/ProjectManager/Scenegraph.js:394]: Instead you can do `const { tree, configurations } = scene`

File: src/assets/scripts/ProjectManager/Scenegraph.js [hunk 2]
```diff
@@ -391,16 +388,22 @@ module.exports = () => {
...
 
     objectControllerkeys.clear();
     sceneprops.configurations.clear();
     sceneprops.config_idx = 0;
 
-    const configs = parseScene(tree, undefined, [], opt);
-    parseSceneConfigurations(configs, undefined, opt);
+    if (/^\d+\.\d+\..+$/.test(sceneprops.project.data.version)) {
```
Comment (mwojcik-sdh) [inline src/assets/scripts/ProjectManager/Scenegraph.js:400]: Can you extract `/^\d+\.\d+\..+$/` to some variable with meaningful name?

**Review by mwojcik-sdh — APPROVED** [review]

File: src/assets/scripts/ProjectManager/Scenegraph.js (hunk 1 above)
Comment (dawidsdh) [reply src/assets/scripts/ProjectManager/Scenegraph.js:394]: It is a es5 script I don't think that we can use it here :D I will leave it as it is. So the cherryGL does not freak out.

File: src/assets/scripts/ProjectManager/Scenegraph.js (hunk 2 above)
Comment (dawidsdh) [reply src/assets/scripts/ProjectManager/Scenegraph.js:400]: I could but the scripts for assets are not modules and we can not import it directly. We have plan to change this scripts to es6 min. and types so we can control what is in this assets.

Comment (mwojcik-sdh) [reply src/assets/scripts/ProjectManager/Scenegraph.js:400]: But I think that variable in the same file will be fine. Just so we know what are we testing here

**Pull request merged**